tests/unit/test_cli.py

```diff
@@ -36,3 +37,15 @@ def test_loops_stable_times(mock_db_cursor):
     wait_for_database(**CLI_PARAMS)
 
     assert mock_db_cursor.call_count == CLI_PARAMS['stable'] + 1
+
+
+@patch('django.db.connection.cursor', side_effect=OperationalError())
+def test_command_error_raised_when_connection_absent(mock_db_cursor):
```

Comment:
Please add a docstring describing the test, e.g.

```suggestion
def test_command_error_raised_when_connection_absent(mock_db_cursor):
    """
    When database connection is absent the management command aborts.
    """
```